packages/reference/src/utils/getProviderName.ts

```diff
@@ -0,0 +1,8 @@
+export function getProviderName(resourceLink: string): string | undefined {
```

Comment:
Can you please add JS doc here with examples of what's expected or some test cases? It's not so obvious for those of us not working directly with the cross-space feature.

Reply:
Test FTW! I added some simple unit tests as documentation.